src/Project/Habitat/code/styles/Habitat/CoveoHabitat.css

```diff
@@ -334,4 +334,79 @@ a.CoveoResultLink:hover,
 .coveo-query-summary-cancel-last{
   color: #c8244f;
   font-weight: 700;
+}
+
+/* Custom loading animation */
+
+.CoveoSearchInterface.coveo-waiting-for-first-query {
+  background-color: #fff;
+}
+
+.CoveoSearchInterface.coveo-waiting-for-first-query .coveo-results-column {
+  display: none;
+}
+
+.loader {
```

Comment:
`loader` est tellement générique, tu peux tu scoper ou ajoute `coveo-` en avant de chacun d'entre eux, genre `coveo-is-loading`